tests/Fixtures/app/console

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env php
```

Comment:
As this console is only used for tests, it should not be present in root dir, but in tests dir at least